src/js/yeldaWebchatInjector.js

```diff
@@ -185,13 +203,14 @@ class YeldaChat {
       this.iframeContainer = document.createElement('div')
       this.iframeContainer.setAttribute('id', 'yelda_iframe_container')
 
-      if (this.parentContainer !== document.body) {
-        // CSS class which contols the opacity and positin of the frame container
+      // Display iframe if webchat should be opened on load, otherwise hide it
+      if (shouldBeOpened) {
+        // CSS class which contols the opacity and position of the frame container
         classList += ' y_active inner'
       } else {
         // If the iframe is inserted into the document body, hide it by default
         // If the webChatContainer is inserted into the document body, the iframeContainer should be hidden it by default
-        // assistant_img click event management will take care of showing and hiding the webchat
+        // yelda_assistant_img click event management will take care of showing and hiding the webchat
         this.iframeContainer.style.cssText = 'display: none;'
       }
```

Comment:
i am not 100% sure about it. 
1. What about assistant creation with "test your chat" button. shouldBeOpened is not supposed to be mandatory set to true, but the iframe should be visible

2. are you sure it's not related to the inner class ?
![image](https://user-images.githubusercontent.com/5340081/88944853-8ca3fb00-d28d-11ea-84a5-10edb262571a.png)

3.hidding

```
if (shouldBeOpened || isStartBtn)  {
 classList += ' y_active'
}

if (this.parentContainer !== document.body) {
        classList += '  inner' // I am not sure about that one, i did not test
}
```
![image](https://user-images.githubusercontent.com/5340081/88944636-464e9c00-d28d-11ea-9e17-752401093127.png)

Reply:
I updated this check because last time I added the following check on the assistant image:
```
// If the webchat should be opened directly, don't display the assistant image
    // This avoids the assistant image to be displayed below the opened webchat instead of being hidden and only appear once the webchat is closed
    if (data.shouldBeOpened) {
      this.assistantImage.style.display = 'none'
    }
```
So because I was checking shouldBeOpened to hide the assistant image, I thought that I should apply the same logic to display the iframe

1. I tested and it works because we set shouldBeOpened to true too but indeed I didn't checked what happened if I removed shouldBeOpened

2. 3. I haven't looked at it but I'll test it !

And the problem with the missing CSS of the default image is actually because we didn't think to update assistant_img to yelda_assistant_img in the CSS file, I'll rename it there

Reply:
And the problem with the missing CSS of the default image is actually because we didn't think to update assistant_img to yelda_assistant_img in the CSS file, I'll rename it there

Indeed we forgot, but the problem is not coming from here, but from a 
```
 Cannot read property 'classList' of null
```
when calling
```
classList.remove('default', 'custom')
```
because the image is not yet added to the DOM.
i'll add a commit fix